app/src/forms/email/assets/status-assignment-email.html

```diff
@@ -1,111 +1,259 @@
+                            </td>
+                          </tr>
 
-    <tr style="height:25pt">
-      <td width="47" valign="top"
-          style="width:35.45pt; border:none; border-left:solid #D9D9D9 1.0pt; background:white; padding:0cm 5.4pt 0cm 5.4pt; height:56.9pt">
-        <p class="MsoNormal" style="margin-bottom:0cm; margin-bottom:.0001pt; line-height:normal">
-          &nbsp;</p>
-      </td>
-      <td width="416" colspan="4" valign="top"
-          style="width:312.05pt; background:white; padding:0cm 5.4pt 0cm 5.4pt; height:56.9pt">
 
-      </td>
-      <td width="55" valign="top"
-          style="width:41.15pt; border:none; border-right:solid #D9D9D9 1.0pt; background:white; padding:0cm 5.4pt 0cm 5.4pt; height:56.9pt">
-        <p class="MsoNormal" style="margin-bottom:0cm; margin-bottom:.0001pt; line-height:normal">
-          &nbsp;</p>
-      </td>
-    </tr>
-    <tr>
-      <td width="47" valign="top"
-          style="width:41.15pt; border:none; border-left:solid #D9D9D9 1.0pt; background:white; padding:0cm 5.4pt 0cm 5.4pt; height:56.9pt">
-        <p class="MsoNormal" style="margin-bottom:0cm; margin-bottom:.0001pt; line-height:normal">
-          &nbsp;</p>
-      </td>
-      <td colspan="4" width="416" valign="top"
-          style="width:41.15pt; border:none; background:white; padding:0cm 5.4pt 0cm 5.4pt; height:56.9pt">
-        <p>
-          <a href="{{ messageLinkUrl }}">{{ messageLinkText }}</a>
-        </p>
-      </td>
-      <td width="55" valign="top"
-          style="width:41.15pt; border:none; border-right:solid #D9D9D9 1.0pt; background:white; padding:0cm 5.4pt 0cm 5.4pt; height:56.9pt">
-        <p class="MsoNormal" style="margin-bottom:0cm; margin-bottom:.0001pt; line-height:normal">
-          &nbsp;</p>
-      </td>
-    </tr>
-    <tr style="height:22.3pt">
-      <td width="518" colspan="6"
-          style="width:388.65pt; border:none; border-top:solid #FCBA19 3.0pt; padding:0cm 5.4pt 0cm 5.4pt; height:22.3pt">
-        <p class="MsoNormal" align="right"
-           style="margin-bottom:0cm; margin-bottom:.0001pt; text-align:right; line-height:normal">
-            <span style="color:white">
-              <a href="https://github.com/bcgov/common-hosted-email-service" target="_blank"
-                 title="Data Source Feedback">
-                <span style="color:white; text-decoration:none">Email powered by BC Gov Common Services</span>
-              </a>
-            </span>
-        </p>
-      </td>
-    </tr>
-    <tr>
-      <td width="59" style="width:44.25pt; padding:0cm 0cm 0cm 0cm"></td>
-      <td width="213" style="width:159.75pt; padding:0cm 0cm 0cm 0cm"></td>
-      <td width="18" style="width:13.5pt; padding:0cm 0cm 0cm 0cm"></td>
-      <td width="241" style="width:180.75pt; padding:0cm 0cm 0cm 0cm"></td>
-      <td width="49" style="width:36.75pt; padding:0cm 0cm 0cm 0cm"></td>
-      <td width="69" style="width:51.75pt; padding:0cm 0cm 0cm 0cm"></td>
-    </tr>
+
+
```

Comment:
Large number of unnecessary whitespace lines. Please make sure to run the VSCode formatter in the right click menu to make sure things get compacted and indented correctly.